include/oneapi/dpl/cmath

```diff
@@ -17,7 +17,9 @@
 #define __ONEDPL_cmath
 
 #include "oneapi/dpl/internal/common_config.h"
+// ATTENTION!!! Include the header from the C++ standard library before the oneDPL config header to see whether the _PSTL_VERSION macro is defined
 #include <cmath>
+#include "oneapi/dpl/pstl/onedpl_config.h"
```

Comment:
Briefly discussed with Ruslan. It looks like that we have "oneapi/dpl/internal/common_config.h" in each public header..
If so then it make sense to add the diagnostic about "C++17" directly in the "common_config.h", and dont include "onedpl_config.h" in everywhere for that.

Reply:
Fixed.